libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKAnalyticsPublisher.h

```diff
@@ -41,4 +43,13 @@ typedef void (^ _Nonnull PublishCompleteBlock)(BOOL success, NSError * _Nullable
  */
 - (void) publish:(nonnull NSArray *) events publishCompleteBlock:(nonnull PublishCompleteBlock) publishCompleteBlock;
```

Comment:
Mark this method deprecated in favor of the one that you have added below. See an example on how to do this [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/dev/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFPasscodeKeyStore.h#L34).

Reply:
Will do